composer.json

```diff
@@ -20,6 +20,7 @@
         "drush/drush": "^11",
         "govcms/govcms": "^2.18",
         "oomphinc/composer-installers-extender": "^2.0",
+        "opis/json-schema": "^2.3",
```

Comment:
JSON Validation library - uses JSON Schema `json-schema.org/` which will work well for our purposes.